trin-core/src/utp/stream.rs

```diff
@@ -367,10 +454,10 @@ pub struct UtpSocket {
     last_rollover: Timestamp,
 
     /// Maximum retransmission retries
-    pub max_retransmission_retries: u32,
+    max_retransmission_retries: u32,
 
     /// Receive channel for discv5 socket
-    rx: Arc<RwLock<mpsc::UnboundedReceiver<Packet>>>,
+    discv_rx: Arc<RwLock<UnboundedReceiver<Packet>>>,
```

Comment:
typo?
```suggestion
    discv5_rx: Arc<RwLock<UnboundedReceiver<Packet>>>,
```

Or just `disc_rx` seems reasonable too.

Reply:
`discv_rx` is still there.

Reply:
Should be fixed now. This slipped around because I renamed it only on line 469.